### DISCUSSION TITLE
How to pre-check on `-dmenu -multi-select`?

### DISCUSSION THREAD

**Discussion by KSXGitHub** [davatorium/rofi#1805] [Ideas/Feature Requests]
When I launch `rofi -dmenu -multi-select`, all boxes are unchecked by default, but my use case requires all boxes are checked by default, and user may manually uncheck undesired boxes.

Questions:
* How to have all boxes checked by default?
* How to have some boxes checked by default?

**Comment by DaveDavenport** [comment]
This is not supported.

**Reply by DaveDavenport** [reply]
What is the (possible) issue?

**Reply by KSXGitHub** [reply]
> What is the (possible) issue?

I am assuming that this tag is for cases when, if the feature is not supported, can be considered a feature request. Or I am misunderstanding things?

**Reply by DaveDavenport** [reply]
For feature request we have the 'feature request/ideas'  category.

**Reply by KSXGitHub** [reply]
I see. I changed it to the correct category.

**Reply by KSXGitHub** [reply]
I created https://github.com/davatorium/rofi/issues/1806. Not sure why the template asks for so many information for a feature request, especially rofi version and configuration.

**Reply by DaveDavenport** [reply]
Because it helps to know the setup of the user when trying to understand what the request actually is.  It is often assumed I know things about their setup, I do not know.

**Comment by yusufaktepe** [comment]
You can swap the symbols for `-ballot-selected-str` and `-ballot-unselected-str`, this will give you a list of unselected items. You can remove them from the list array afterward.

**Reply by KSXGitHub** [reply]
This workaround is not without downsides:
* It makes the code complicated.
* It cannot solve the case where only some are pre-checked.
